miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -194,15 +207,62 @@ Sample.hot = {
     var classes = Hot.sortByProperty(Hot.sampleOptions.sampleClassesDtos, 'id');
     select.push('<select id="classDropdown">');
     select.push('<option value="">Select class</option>');
-    for (var i=0; i<classes.length; i++) {
-      if (classes[i].alias == "Identity") continue;
-      select.push('<option value="'+ classes[i].id +'">'+ classes[i].alias +'</option>');
+    var classOptions = Sample.hot.getNewSampleClassOptions();
+    for (var i=0; i<classOptions.length; i++) {
+     if (classes[i].alias == "Identity") continue;
```

Comment:
This line should be removed. It's looking at the wrong array (classes) and ends up omitting a sample class that should be included. The array it should be looking at (classOptions) is already filtered